.devcontainer/cpp/test/testsuite.bats

```diff
@@ -205,3 +205,15 @@ function build_and_run_with_sanitizers() {
   assert_failure
   assert_output --partial "runtime error: load of null pointer"
 }
+
+@test "using clang-uml to generate uml class diagram" {
+  run cmake --preset clang
+  assert_success
```

Comment:
Remove the `run` and `assert_success`, every step that fails will fail the test.

See: https://bats-core.readthedocs.io/en/stable/writing-tests.html#when-not-to-use-run

Reply:
Well it initially made sense to have run but in anyway clang uml will fail regardless. So I removed it. Thanks